packages/scaffold-config/src/frameworks.ts

```diff
@@ -17,11 +17,14 @@ export type CodeGenFramework = Cypress.ResolvedComponentFrameworkDefinition['cod
 export async function isDependencyInstalled (dependency: Cypress.CypressComponentDependency, projectPath: string): Promise<Cypress.DependencyToInstall> {
   try {
     debug('detecting %s in %s', dependency.package, projectPath)
-    const loc = require.resolve(path.join(dependency.package, 'package.json'), {
-      paths: [projectPath],
-    })
 
-    const pkg = await fs.readJson(loc) as PkgJson
+    const packageFilePath = resolvePackagePath(dependency.package, projectPath) || ''
```

Comment:
Does this also work as expected for nested modules like `@vitejs/plugin-react` etc?

Reply:
yeah, I tested it with a different project
```
> resolvePackagePath('@vitejs/plugin-react', resolve('.'))
'/Users/adams/projects/vite-react-17/node_modules/@vitejs/plugin-react/package.json'
```